module-1/pipelines-project/your-code/readme_miriam.md

```diff
@@ -0,0 +1,96 @@
+# Pipeline Project
```

Comment:
It's not necessary to replicate your code in the readme doc. It's more important that you describe what you want to do, what's the rational behind it and what's is your best approach to solve it.